engine/src/main/java/org/terasology/rendering/logic/MeshRenderer.java

```diff
@@ -244,6 +242,21 @@ private void renderEntitiesByMaterial(SetMultimap<Material, EntityRef> meshByMat
         }
     }
 
+    /**
+     * Checks whether the entity at the given position is relevant.
+     * <p>
+     * The entity is at the given position is relevant if
+     * a) the entity itself is always relevant, or
+     * b) if the block at the position is relevant.
```

Comment:
I'd probably also remove the ", or" one line above, but I now understand better how you tried to let it read like a whole sentence. It's up to you.